analyzers/src/SonarAnalyzer.CSharp/Rules/NotAssignedPrivateMember.cs

```diff
@@ -194,15 +194,15 @@ private static ISet<ISymbol> GetAssignedMemberSymbols(IList<MemberUsage> memberU
 
                 if (PreOrPostfixOpSyntaxKinds.Contains(parentNode.Kind()))
                 {
-                    assignedMembers.Add(memberSymbol);
+                    AddAssignedMember(assignedMembers, memberSymbol);
                     continue;
                 }
 
                 if (parentNode is AssignmentExpressionSyntax assignment)
                 {
                     if (assignment.Left == node)
                     {
-                        assignedMembers.Add(memberSymbol);
+                        AddAssignedMember(assignedMembers, memberSymbol);
```

Comment:
These two nested `if` can be safely collapsed.

Reply:
Well, the whole thing seems simplifiable into a state where we won't need the method/local function. Just some `if a || b || c then add`

Reply:
Happy to eliminate that one, indeed :)